static/js/Autocomplete.jsx

```diff
@@ -13,7 +13,18 @@ const _type_icon_map = {
   "ref": "iconmonstr-book-15.svg",
   "search": "iconmonstr-magnifier-2.svg",
   "Term": "iconmonstr-script-2.svg",
-  "User": "iconmonstr-user-2%20%281%29.svg"
+  "User": "profile.svg"
+};
+const type_title_map = {
```

Comment:
This map should be unnecessary. It can just contain a mapping from type to the English. The Hebrew can be translated via `<InterfaceText>` using the following syntax `<IntefaceText>{en}</InterfaceText>`. This will pull out the correct Hebrew from `strings.js`. Confirm that the correct translation for each of these categories exists in `strings.js`.